### ISSUE TITLE
Feature request: Pre-scheduled send event

### ISSUE THREAD

**Issue by Roensby** [InboxSDK/InboxSDK#1225]
It would be great to have a compose event for when the user clicks the "Schedule send" button. My use case is to be able to modify the email before it is being scheduled, as I would like to treat scheduling as another way of "sending" an email.

Like so:

```js
sdk.Compose.registerComposeViewHandler((composeView) => {
  composeView.on('prescheduledsending', (event) => {
    // ...
  })
})
```

**Comment by Roensby** [comment]
This pr https://github.com/InboxSDK/InboxSDK/issues/1225 adds the requested functionality, but doesn't close the submenu that appears when the user clicks the "More send options" button next to the "Send" button.

**Comment by Roensby** [comment]
I've added a few features to this PR that should improve the usefulness of the prescheduledsend event.
Specifically, I've added

- Also listens for keyboard event 'enter' if the user navigates the 'More send options' dropdown menu with keyboard.
- Includes a `composeView.scheduleSend()` method that allows to call the schedule email modal at anytime.
```js
sdk.Compose.registerComposeViewHandler((composeView) => {
  composeView.on('prescheduledsending', (event) => {
    event.cancel()
    // Do stuff here...
    composeView.scheduleSend()
  })
})
```
- Filters out non-trusted events to avoid having an infinite prescheduledevent loop.
